src/post_processors/AbstractPostProcessor.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
```

Comment:
Refactor file name pls

Reply:
What would like it be called?

Reply:
abstract_post_processor.py

Reply:
Maybe we can skip PostProcessor part in file name

Reply:
Sure, I usually name such modules base.py

Reply:
Okay, I think we can avoid even underscores, so let call it just src/post_processors/abstract.py